package.json

```diff
@@ -38,14 +38,14 @@
     "chai": "^4.2.0",
     "decimal.js": "^10.2.1",
     "ethereum-waffle": "^3.0.2",
-    "ethers": "^5.0.8",
+    "ethers": "5.0.24",
     "hardhat": "^2.2.0",
     "hardhat-typechain": "^0.3.5",
     "mocha": "^6.2.2",
     "mocha-chai-jest-snapshot": "^1.1.0",
     "prettier": "^2.0.5",
     "prettier-plugin-solidity": "^1.0.0-alpha.59",
-    "@tenderly/hardhat-tenderly": "file:../../../../hardhat-tenderly-v1",
+    "@tenderly/hardhat-tenderly": "file:../../../../hardhat-tenderly-v1/packages/tenderly-hardhat",
```

Comment:
A beta package version should be released for the purpose of uniswap tests shooting that specially released `@tenderly/hardhat-tenderly`.

We need to do this because we downgraded `@tenderly/hardhat-tenderly` to use `ethers@5.0.8` on that special release so it is compatible to the uniswap tests.